api/class-wc-calypso-bridge-settings-email-groups-controller.php

```diff
@@ -0,0 +1,107 @@
+<?php
+/**
+ * REST API Settings controller
+ *
+ * Handles requests to the /settings/batch_email endpoint.
+ *
+ * @author   Automattic
+ * @category API
+ * @package  WooCommerce/API
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * REST API Settings controller class.
+ *
+ * @package WooCommerce/API
+ */
+class WC_Calypso_Bridge_Settings_Email_Groups_Controller extends WC_REST_Settings_Controller {
+
+	/**
+	 * WP REST API namespace/version.
+	 */
+	protected $namespace = 'wc/v3';
+	protected $rest_base = 'settings_email_groups';
+
+	/**
+	 * Register routes.
+	 *
+	 */
+	public function register_routes() {
+		register_rest_route( $this->namespace, '/' . $this->rest_base, array(
+			array(
+				'methods'             => WP_REST_Server::READABLE,
+				'callback'            => array( $this, 'get_all_email_settings' ),
+				'permission_callback' => array( $this, 'get_items_permissions_check' ),
+			),
+		) );
+	}
+
+	/**
+	 * Bulk read email setings;
+	 *
+	 * @return array Of WP_Error or WP_REST_Response.
```

Comment:
From the `WC_REST_Setting_Options_Controller` code it looks like they use `@return WP_Error|WP_REST_Response`